application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -46,16 +46,36 @@ Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interva
 Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
 
+#Config
+Analytics.Code.ConfigurationClass.matomoTrackingCode.hint=Tracking code received from Matomo. The 'u' variable might be different than the one that is initially generated by Matomo if the server you want to track is on another machine. To fix this, go to the integration tab, copy the Matomo URL, and replace the value of 'u' with the Matomo URL.
+Analytics.Code.ConfigurationClass.idSite.hint=The ID of the site that you want to see the statistics for (found in the integration tab).
+Analytics.Code.ConfigurationClass.requestAddress.hint=The Matomo url (found in the integration tab).
+Analytics.Code.ConfigurationClass.authToken.hint=Token generated by Matomo to be capable of using the API to view the data.
+
+#Most Viewed Pages 
+mostViewedPages.label=Page Title
+mostViewedPages.nb_visits=Unique Vistators
+mostViewedPages.nb_hits=Page Views
+mostViewedPages.sum_time_spent=Time spent(s)
+mostViewedPages.entry_nb_visits=No. of views that started on this page
+mostViewedPages.entry_nb_actions= No. Of Actions
+mostViewedPages.bounce_rate=Bounce Rate
+mostViewedPages.exit_rate=Exit Rate
+
 # Live table keys
 analytics.livetable.doc.title=Page Title
-analytics.livetable.startDate = Start Date
-analytics.livetable.endDate = End Date
+analytics.livetable.startDate=Start Date
+analytics.livetable.endDate=End Date
 analytics.livetable.doc.name=Page Name
 analytics.livetable.emptyvalue=Not Defined
 
+#Admin Section
```

Comment:
This is also a change added by mistake https://github.com/xwikisas/application-analytics/pull/6/files#diff-38492de9bb01b5aebc39fbc8e135b86022a162eccda2948b8021381a82f03099
Using the `git add -p` would be helpful for these cases